test/integration/puppetlabs/services/jruby_pool_manager/jruby_pool_int_test.clj

```diff
@@ -173,63 +148,36 @@
      (let [config (jruby-testutils/jruby-config {:max-active-instances 4
                                                  :borrow-timeout test-borrow-timeout})
            pool-manager-service (tk-app/get-service app :PoolManagerService)
-           pool-context (pool-manager-protocol/create-pool pool-manager-service config)]
+           pool-context (pool-manager-protocol/create-pool pool-manager-service config)
+           pool-state (jruby-internal/get-pool-state pool-context)
+           pool (:pool pool-state)]
```

Comment:
Doesn't seem like `pool-state` is being used, so I think you could just use `jruby-internal/get-pool`.